TAT.homework5_2/TableBuilder.java

```diff
@@ -0,0 +1,48 @@
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Provides creating of table containing list of servers with response delays
+ * @author Yury Suponev
+ */
+public class TableBuilder {

```

Comment:
I would say it should be closer to implementation - at least ServerResponcesTableBUilder